src/graphql/data/util.tsx

```diff
@@ -201,7 +201,7 @@ export const BACKEND_SUPPORTED_CHAINS = [
   Chain.Bnb,
   Chain.Celo,
 ] as const
-export const BACKEND_NOT_YET_SUPPORTED_CHAIN_IDS = [ChainId.AVALANCHE] as const
+export const BACKEND_NOT_YET_SUPPORTED_CHAINS = [ChainId.AVALANCHE] as const
```

Comment:
why was this renamed? In my mind `chains` refers to the BE chain name strings where as chainId is specifically the number representation